### PR TITLE
r/glue_catalog_database - add arn + disappears test

### PR DESCRIPTION
**Pull request by DrFaust92**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates #13624, #13527, #13820

Release note for [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note
resource_aws_glue_catalog_database: add arn attribute
```

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'
--- PASS: TestAccAWSGlueCatalogDatabase_full (101.79s)
--- PASS: TestAccAWSGlueCatalogDatabase_disappears (28.91s)
```

### CODE REVIEW

File: aws/resource_aws_glue_catalog_database.go [hunk 1]
```diff
@@ -141,6 +146,15 @@ func resourceAwsGlueCatalogDatabaseRead(d *schema.ResourceData, meta interface{}
 		return fmt.Errorf("Error reading Glue Catalog Database: %s", err.Error())
 	}
 
+	databaseArn := arn.ARN{
+		Partition: meta.(*AWSClient).partition,
+		Service:   "glue",
+		Region:    meta.(*AWSClient).region,
+		AccountID: meta.(*AWSClient).accountid,
```
[the remark this answers is not in the document]
Comment (bflad) [reply aws/resource_aws_glue_catalog_database.go:153]: https://github.com/terraform-providers/terraform-provider-aws/pull/13456 has been merged so this should be safe to add now 👍

[the remark this answers is not in the document]
Comment (DrFaust92) [reply aws/resource_aws_glue_catalog_database.go:153]: done

File: website/docs/r/glue_catalog_database.html.markdown [hunk 2]
```diff
@@ -28,6 +28,13 @@ The following arguments are supported:
 * `location_uri` - (Optional) The location of the database (for example, an HDFS path).
 * `parameters` - (Optional) A list of key-value pairs that define parameters and properties of the database.
 
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - Catalog ID and name of the database
+* `arn` - The ARN of the Glue Connection.
```
Comment (bflad) [inline website/docs/r/glue_catalog_database.html.markdown:36]:
```suggestion
* `arn` - The ARN of the Glue Catalog Database.
```

**Comment by DrFaust92** [comment]
added change for #13820

**Pull request merged**